pkg/services/rendering/rendering.go

```diff
@@ -110,6 +111,19 @@ func (rs *RenderingService) RenderErrorImage(err error) (*RenderResult, error) {
 }
 
 func (rs *RenderingService) Render(ctx context.Context, opts Opts) (*RenderResult, error) {
+	startTime := time.Now()
```

Comment:
Maybe need to handle timeout error here? Or at least a defer to make sure failed/completed/elapsed time metrics are increased before return.

Reply:
Not sure what you mean. `rs.render()` returns error if timeout exceeded. So metrics will be evaluated.

Reply:
Yeah you probably right. But if the error is "context deadline exceeded" we may want to set status label as "cancelled" instead of "error" for metrics?